### PR TITLE
Setup default genesis ledger

### PR DESCRIPTION
**Pull request by nholland94**

### CODE REVIEW

File: genesis_ledgers/default.json [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+  "ledger": {
+     "name": "testnet_postake"
+  }
```
Comment (lk86) [inline genesis_ledgers/default.json:4]: Is this enough for the daemon to start?

**Review by lk86 — APPROVED** [review]

File: genesis_ledgers/default.json (hunk 1 above)
Comment (nholland94) [reply genesis_ledgers/default.json:4]: Yes, it should be. I'm testing to be sure before I merge, but everything else should just default to the compile time constants. This ledger name maps to a precompiled ledger that is available to all daemons.

Comment (nholland94) [reply genesis_ledgers/default.json:4]: Tested locally, and appears to work fine.

**Comment by aneesharaines** [comment]
!approved-for-mainnet

**Pull request merged**